docs/configuration/http_conn_man/http_conn_man.rst

```diff
@@ -141,6 +141,11 @@ http2_settings
     window. Currently , this has the same minimum/maximum/default as :ref:`initial_stream_window_size
     <config_http_conn_man_http2_settings_initial_stream_window_size>`.
 
+  per_stream_buffer_limit_bytes
```

Comment:
This doesn't exist anymore right? Should this text get merged into initial_stream_window_size?